Formula/grype.rb

```diff
@@ -0,0 +1,35 @@
+class Grype < Formula
+  desc "Vulnerbility scanner for container images and filesystems"
+  homepage "https://github.com/anchore/grype"
+  url "https://github.com/anchore/grype/archive/refs/tags/v0.62.3.tar.gz"
+  sha256 "f97c8412f7c5f74a348fe011f84f5926153d23475d875e2dc61ddca30872e489"
+  license "Apache-2.0"
+  head "https://github.com/anchore/grype.git", branch: "main"
+
+  depends_on "go" => :build
+
+  def install
+    ldflags = %W[
+      -s -w
+      -X github.com/anchore/grype/internal/version.version=#{version}
+      -X github.com/anchore/grype/internal/version.gitCommit=brew
+      -X github.com/anchore/grype/internal/version.buildDate=#{time.iso8601}
+    ]
+
+    # Building for OSX with -extldflags "-static" results in the error:
+    # ld: library not found for -lcrt0.o
+    # This is because static builds are only possible if all libraries
```

Comment:
If all libraries what? And why do we want static?

Reply:
yeah, trying to mimic what upstream does, I can remove it.